File: src/lec08/Lec08Main.java
package lec08;

public class Lec08Main {

    public static void main(String[] args) {

    }

    public int max(int a, int b) {
        if (a > b) {
            return a;
        }
        return b;
    }

    public static void printAll(String... strings){
        for (String string : strings) {
            System.out.println(string);
        }
    }
}

File: src/lec08/Lec08Main.kt
package lec08


fun main(){
    println(max(2, 3))
    repeat("Hello",4 , useNewLine = false)
}

private fun max(a: Int, b: Int) = if (a > b) a else b

fun repeat(
    str : String,
    num : Int = 3,
    useNewLine : Boolean = true
){
    for(i in 1..num){
        if(useNewLine){
            println(str)
        }else{
            println(str)
        }
    }
}